packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -123,67 +123,76 @@ const COLUMNS = [
         vms: getAllVms(state, props),
         vbds: getVbds(state, props),
       })
-    })(({ vbds, vms }) => {
-      if (isEmpty(vms)) {
+    })(({ item: vdi, vbds, vms, userData: { vmsSnapshotsBySuspendedVdi } }) => {
+      const vmSnapshot = vmsSnapshotsBySuspendedVdi[vdi.uuid]?.[0]
+      if (isEmpty(vms) && vmSnapshot === undefined) {
```

Comment:
I don't think it's necessary to test `vmSnapshot === undefined` as I think the test is only there to make sure objects have been fetched.